ingeniamotion/communication.py

```diff
@@ -225,6 +227,50 @@ def connect_servo_eoe_service_interface_ip(
             net_status_listener,
         )
 
+    def connect_servo_comkit(
+        self,
+        ip: str,
+        coco_dict_path: str,
+        moco_dict_path: str,
+        alias: str = DEFAULT_SERVO,
+        port: int = 1061,
+        connection_timeout: int = 1,
+        servo_status_listener: bool = False,
+        net_status_listener: bool = False,
+    ) -> None:
+        """Connect to target servo using a COM-KIT
+
+        Args:
+            ip : servo IP
+            coco_dict_path : COCO dictionary path.
+            moco_dict_path : MOCO dictionary path.
+            alias : servo alias to reference it. ``default`` by default.
+            port : servo port. ``1061`` by default.
+            connection_timeout: Timeout in seconds for connection.
+                ``1`` seconds by default.
+            servo_status_listener : Toggle the listener of the servo for
+                its status, errors, faults, etc.
+            net_status_listener : Toggle the listener of the network
+                status, connection and disconnection.
+
+        Raises:
+            FileNotFoundError: If a dict file doesn't exist.
+            ingenialink.exceptions.ILError: If the servo's IP or port is incorrect.
+        """
+        for dict_path in [coco_dict_path, moco_dict_path]:
+            if not path.isfile(dict_path):
+                raise FileNotFoundError(f"{dict_path} file does not exist!")
+        self.__comkit_dict = create_comkit_dictionary(coco_dict_path, moco_dict_path)
```

Comment:
The comkit dictionary should not be a class property. You can simply remove the file in the 'connect_servo_comkit' function after "__servo_connect" call

Reply:
Done!